gbn/queue.go

```diff
@@ -60,7 +60,7 @@ func newQueue(s uint8, handshakeTimeout time.Duration,
 		content:          make([]*PacketData, s),
 		s:                s,
 		handshakeTimeout: handshakeTimeout,
-		log:              logger,
+		log:              log,
```

Comment:
I believe this change was ment to go into the previous commit?

Reply:
indeed :)